keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/GitMetadata.kt

```diff
@@ -1,9 +1,13 @@
 package com.netflix.spinnaker.keel.api.artifacts
 
 /**
- * todo eb: other information should go here, like a link to the commit. But that needs to be done
- * in a scalable way. For now, this is just a minimal container for information we can parse from the version.
+ * The git metadata of an artifact.
  */
 data class GitMetadata(
-  val commit: String
+  val commit: String, // commit hash, can be short or long sha
+  val author: String? = null,
+  val linkToCommit: String? = null,
+  val repoName: String? = null, // the repository name, like "myApp"
+  val projectName: String? = null, // the project name, like SPKR
+  val commitMessage: String? = null
```

Comment:
this could get long, we will need a limit to what we store here

Reply:
Ya, I think we should limit because we don't want any SQL errors like "overflow" or whatever it gives us. We can just truncate the commitMessage before we save it